.github/workflows/build_docs.yml

```diff
@@ -0,0 +1,76 @@
+name: Build and Deploy Documentation
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'docs/**'
+      - '.github/workflows/build_docs.yml'
+  pull_request:
+    paths:
+      - 'docs/**'
+      - '.github/workflows/build_docs.yml'
+  workflow_dispatch:
+
+jobs:
+  build_docs:
+    name: Build Mesh Adaptation Documentation
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/mesh-adaptation/firedrake-um2n:latest
+      options: --user root
+      volumes:
+        - ${{ github.workspace }}:/home/firedrake/output
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install Animate and Goalie
+        run: |
+          . /home/firedrake/firedrake/bin/activate
+          cd /home/firedrake/firedrake/src/
+          git clone https://github.com/mesh-adaptation/animate.git
+          git clone https://github.com/mesh-adaptation/goalie.git
+          pip install -e animate
+          pip install -e goalie
+        working-directory: /home/firedrake/output/
+        shell: bash
```

Comment:
How come you only install two of the packages?

Reply:
Because Movement and UM2N are already installed in the UM2N container :)

Reply:
Ah okay. It might be worth uninstalling and then reinstalling them, though. Just in case the Docker build is out of date.

Reply:
Ah good point! Thanks! Done in 9f29769edb9c6bdd0dc5cf320d7b4a7935dc2711 and a220a40ff1f6c4de9fc47a7d9a74333252532ca4